Doc/howto/clinic.rst

```diff
@@ -1408,7 +1408,7 @@ Let's start with defining some terminology:
   ``two-pass``
     A buffer like ``buffer``.  However, a two-pass buffer can only
     be written once, and it prints out all text sent to it during
-    all of processing, even from Clinic blocks *after* the
+    all of processing, even from Clinic blocks *after* the current one.
```

Comment:
My guess is @larryhastings meant after the dumping point.

_Of_ should also be removed from “during all [of] processing”.

And would it be better to say “a two-pass buffer can only be _dumped_ once”? Otherwise it sounds like you can only use the two-pass destination once, and then you have to use some other destination.